app/battle/knight_creator.py

```diff
@@ -0,0 +1,85 @@
+class Knight:
+
+    def __init__(
+            self,
+            knight_nik: str,
```

Comment:
Don't need to add `knight_` part everywhere. It is easy to understand, because it is Knight object attribute
Also, from the description you know that here u use `name`, not a `nickname`. So change that